src/App.js

```diff
@@ -1,18 +1,16 @@
 import React, { useState, useEffect } from 'react';
 import { Route, Link, Switch } from 'react-router-dom';
```

Comment:
I saw the Link component isn't being used anymore. Should be fine to remove from the imports.

Reply:
Good looking out. I dont know if we will need it in the future but for now it would be fine to remove. One less thing yelling at us for not being in use.

Reply:
removed Link!